src/test/java/org/alfresco/repo/web/scripts/servlet/RemoteAuthenticatorFactoryAdminConsoleAccessTest.java

```diff
@@ -198,6 +212,35 @@ public void testAdminCanAccessAdminConsoleHelperScript()
         complexCheckOfScriptCases(families);
     }
 
+    /**
+     * Tested access to the AdminConsole for an non literal admin user
```

Comment:
What do we mean by "non literal admin user" ? I notice it is also in the title of this PR but not mentioned in the linked JIRAs (REPO &/or MNT) ?

Is this new terminology for an existing concept, eg. default "admin" user (bearing in mind that the latter is configurable prior to bootstrap, hence could be "xyz123").

Reply:
It is any user added to "ALFRESCO_ADMINISTRATORS" group, not just "admin".

Reply:
This expression was picked from: https://github.com/Alfresco/alfresco-remote-api/pull/231, which is related to this ticket in somehow